packages/eth-providers/src/base-provider.ts

```diff
@@ -554,11 +573,9 @@ export abstract class BaseProvider extends AbstractProvider {
   isReady = async (): Promise<void> => {
     try {
       await this.api.isReadyOrError;
-      await this.getNetwork();
```

Comment:
`getNetwork` is only here, but it has been deleted. Is it no longer needed?

Reply:
it's moved to here https://github.com/AcalaNetwork/bodhi.js/pull/902/files#diff-1dc05f5efb9f973bee2a8da85d8839f8f96514217f7336e218b5c2535bd20baeR609

the order was not correct before, since we usually check `get_network` to make sure the rpc is ready. So we should check `api.isReady` in `getNetwork`, not the opposite